fastlane/lib/fastlane/actions/apple_pay_cert.rb

```diff
@@ -0,0 +1,153 @@
+            raise ex
+          end
+        end
+
+        x509_certificate = cert.download
+        filename_base = params[:merchant_bundle_id] # PEM.config[:pem_name] || "#{PEM.config[:app_identifier]}"
+        filename_base = File.basename(filename_base, ".pem") # strip off the .pem if it was provided.
+
+        output_path = File.expand_path('.')
+        FileUtils.mkdir_p(output_path)
+
+        if params[:save_private_key]
+          private_key_path = File.join(output_path, "#{filename_base}.pkey")
+          File.write(private_key_path, pkey.to_pem)
+          UI.message("Private key: ".green + Pathname.new(private_key_path).realpath.to_s)
+        end
+
+        if params[:generate_p12]
+          p12_cert_path = File.join(output_path, "#{filename_base}.p12")
+          p12 = OpenSSL::PKCS12.create(params[:p12_password], 'production', pkey, x509_certificate)
+          File.write(p12_cert_path, p12.to_der)
+          UI.message("p12 certificate: ".green + Pathname.new(p12_cert_path).realpath.to_s)
+        end
+
+        x509_cert_path = File.join(output_path, "#{filename_base}.pem")
+        File.write(x509_cert_path, x509_certificate.to_pem + pkey.to_pem)
+        UI.message("PEM: ".green + Pathname.new(x509_cert_path).realpath.to_s)
+        return x509_cert_path
+      end
+
+      def self.certificate
+        Spaceship.certificate.apple_pay_certificate
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.is_supported?(platform)
+        [:ios].include?(platform)
+      end
+
+      def self.description
+        'Creates Apple Pay certificate'
+      end
+
+      def self.details
+        "This action allows you to create Apple Pay merchant certificate."
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :username,
+                                       env_name: "APPLE_PAY_USERNAME",
+                                       description: "Your Apple ID Username"),
+          FastlaneCore::ConfigItem.new(key: :merchant_bundle_id,
+                                   env_name: "APPLE_PAY_MERCHANT_BUNDLE_ID",
+                                   description: "You merchant bundle identifier (e.g: merchant.com.example)"),
+          FastlaneCore::ConfigItem.new(key: :team_id,
+                                 short_option: "-b",
+                                 env_name: "APPLE_PAY_CERT_TEAM_ID",
+                                 code_gen_sensitive: true,
+                                 default_value: CredentialsManager::AppfileConfig.try_fetch_value(:team_id),
+                                 default_value_dynamic: true,
+                                 description: "The ID of your Developer Portal team if you're in multiple teams",
+                                 optional: true,
+                                 verify_block: proc do |value|
```

Comment:
what does this verify block do an why?

Reply:
still unanswered. 

I think this block this way is usually used to just copy the value to the ENV var, but I am unsure if this is actually what we want to do here as `APPLE_PAY_CERT_TEAM_ID != FASTLANE_TEAM_ID`

Reply:
I think `APPLE_PAY_CERT_TEAM_ID` is the `FASTLANE_TEAM_ID`. Why do you think it's not?

Reply:
Do we want to set the globally applied ENV var via an action that "only" creates the apple pay cert?

(ping @mollyIV: Another usage of ENV vars that is a bit strange...)

Reply:
unanswered